pkg/neorpc/result/version.go

```diff
@@ -29,6 +37,8 @@ type (
 		MemoryPoolMaxTransactions   int
 		ValidatorsCount             byte
 		InitialGasDistribution      fixedn.Fixed8
+		// Hardforks is the list of network hardforks sorted by name.
+		Hardforks []Hardfork
```

Comment:
I'm not sure, is it better to keep the same structure as in C# or to pack them in map. List allows to keep them sorted while map allows quick existence check.

Reply:
Although in C# client it's a map.

Reply:
Map should be OK for it.

Reply:
Fixed.